.jenkins/Jenkinsfile

```diff
@@ -211,7 +214,11 @@ def win2016ReleaseCrossCompile() {
         node('SGXFLC-Windows') {
             cleanWs()
             checkout scm
-            PowerShellWrapper('./scripts/test-build-config.ps1 -build_type Release -build_enclaves')
+            dir('build') {
```

Comment:
The builds should occur in the build/X64-Release directory for the Release Cross Compile with ninja.

Reply:
Same as above.

Reply:
Done!